Gemfile

```diff
@@ -18,6 +18,8 @@ gem 'okcomputer' # for monitoring
 gem 'rubocop', group: [:development, :test]
 gem 'rubocop-rspec', group: [:development, :test]
 
+gem 'rest-client'
```

Comment:
Can we use `faraday` instead of `rest-client`? We've been trying to migrate off rest-client after it went through a long period without maintenance, and `faraday` is generally more amenable to features like persistent http connections, http stubbing in tests ,etc.